src/trust.rs

```diff
@@ -103,11 +118,13 @@ pub fn create_trust_anchor(
     log::debug!("Private key extracted.");
 
     // Private key printed to terminal, when keyout argument not specified.
-    match keyout {
-        Some(file_name) => write_to_file(file_name, &private_key, "App private key"),
-        _ => {
-            println!("Private key for an application is used to sign device certificates, see `drg trust add --help`\n");
-            println!("{}", &private_key)
+    if !is_input_key {
```

Comment:
```suggestion
    if key_input.is_None(); {
```

Reply:
This leads `value borrowed here after move` error

Reply:
Ha ok I see why, sorry i did not catch it earlier